arduino/opencr_arduino/opencr/libraries/turtlebot3/turtlebot3_waffle/turtlebot3_core/turtlebot3_core.ino

```diff
@@ -230,6 +268,7 @@ void loop()
 
   // Call all the callbacks waiting to be called at that point in time
   nh.spinOnce();
+  delay(10);
```

Comment:
Can you put a comment in here as to why it's necessary. Otherwise it looks like a magic number.